src/index.html

```diff
@@ -11,12 +11,100 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul class="links-list">
+          <li class="nav__link">
+            <a
+              class="link is-active"
+              href="index.html"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__link">
```

Comment:
Missing empty lines:

You'd missed the empty lines that should go between sibling blocks. Check the [Code Styles Rules](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) it's on HTML Formatting